src/app/blog/index.html

```diff
@@ -3,6 +3,29 @@
   pageTitle="SKY UX Blog"
   windowTitle="SKY UX Blog">
 
+	<article class="blog-post">
+		<header>
+			<h2 class="sky-section-heading">Design update — November 2018</h2>
+			<p>SKY UX releases 2.28.1 - 2.31.0</p>
+		</header>
+		<section>
+			<h3 class="sky-subsection-heading">Recently completed</h3>
+				<h4 class="sky-emphasized">Bugs</h4>
+					<p>We spent a lot of time this month squashing bugs. Check the <a href="https://github.com/blackbaud/skyux2/blob/master/CHANGELOG.md">changelog</a> for the full list.</p>
+		</section>
+		<section>
+			<h3 class="sky-subsection-heading">In the works</h3>
+			<h4 class="sky-emphasized"><a href="https://github.com/blackbaud/skyux-grids/pull/16">Grid</a></h4>
+					<p>We are adding multiselect capability to grid.</p>
+			<h4 class="sky-emphasized"><a href="https://github.com/blackbaud/skyux-layout/pull/5">Summary action bar</a></h4>
+					<p>We are porting the summary action bar from SKY UX 1.</p>
+			<h4 class="sky-emphasized"><a href="https://github.com/blackbaud/skyux-modals/pull/7">Modal</a></h4>
+					<p>We are adding an event that triggers before a modal closes to support displaying a confirmation dialog in case there are unsaved changes.</p>
```

Comment:
It's minor, but I'd delete "displaying" and change "in case there are unsaved changes" to "when the modal has unsaved changes."